articles/provider-1qbit.md

```diff
@@ -17,20 +17,20 @@ uid: microsoft.quantum.providers.optimization.1qbit
 
 ## Targets
 
-### Tabu Search Solver 
+### Tabu Search Solver
 An iterative heuristic algorithm that uses local search techniques to solve a QUBO problem. It starts from a random solution and looks for an improved solution in the solution's neighborhood which includes all possible single flips. The algorithm stops when it reaches a stopping criterion, such as a specified number of consecutive iterations without improvement.
 
-****** 
-Please note for the parameters values, they are required to be of JSON string type.  The types listed below are the types the solvers expect within the string value.  For example, 'improvement_cutoff' is listed as type int, it is expected to be passed in in the format:
-
-improvement_cutoff: "5"
-******
-
 - Job type: `Quantum-Inspired Optimization Problem`
 - Data Format: `microsoft.qio.v2`
 - Target ID: `1qbit.tabu`
 - Python Solver class name: `TabuSearch`
 
+******
```

Comment:
@dime10 what's the suggested formatting here?

Reply:
Proper format for the notes is: (already updated for the first instance in main from a previous PR)

>[NOTE!]
> Parameter values are required to be of JSON string type.  The types listed here are the types that the solvers expect within the string value.  For example, `improvement_cutoff` is listed as type int, and is expected to be passed in the format:
>
>**improvement_cutoff: "5"**